app/emails/applications.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from app.models import Application
+from flask import current_app, url_for
+from dmutils.email import send_email, EmailError
+
+from .rendering import render_email_template
+
+
+def send_approval_notification(application_id):
+    TEMPLATE_FILENAME = 'application_approved.md'
+
+    FRONTEND_ADDRESS = current_app.config['FRONTEND_ADDRESS']
```

Comment:
Well, it's a bit different because it's not the server name of the app itself, it's the name of the frontend